source/llassetgen/source/Packing.cpp

```diff
@@ -16,43 +26,33 @@ namespace llassetgen {
             }
 
             if (usedHeight + rectSize.y > packing.atlasSize.y) {
-                return false;
+                if (allowGrowth) {
+                    PackingSizeType finalHeight = usedHeight + rectSize.y;
+                    auto numDoublings = ceilLog2(ceilDiv(finalHeight, packing.atlasSize.y));
```

Comment:
Given `predictAtlasSize`'s behavior, this should always be 1, right?

Reply:
Without rotations yes, with rotations I'm not entirely sure (`predictAtlasSize` does not take rotations into account). I'd rather be defensive here and leave it as is (also allows changes to how we calculate the atlas size without changes being needed here).